python-checks/src/test/resources/checks/hotspots/debugMode/settings.py

```diff
@@ -2,4 +2,12 @@
 DEBUG_PROPAGATE_EXCEPTIONS = True  # Noncompliant
 DEBUG += True  # OK
 DEBUG = False  # OK
+DEBUG = print()  # OK
+DEBUG_PROPAGATE_EXCEPTIONS = False  # OK
 Other = True
+
+class MyClass:
+    def __init__(self):
+        self.boolean_property = True
+my_object = MyClass()
+my_object.boolean_property = True
```

Comment:
Not sure exactly what is tested here, wouldn't this be covered by the `Other` case already?

Reply:
This is just to increase the coverage. It is important to have an fqn which corresponds to a boolean property on the lhs. This is to fix the low coverage that occurred on the existing code. 

Since we want our test files to only have valid python code, I decided it would be simplest to create this class for the test.